ui/apps/platform/src/Containers/Login/TestLoginResultsPage.js

```diff
@@ -56,6 +56,12 @@ function getMessage(response) {
                 <h2 className="font-700">User Attributes:</h2>
                 <ul className="list-none">{displayAttributes}</ul>
             </p>
+            <p className="pb-2 mb-2 border-b border-success-700">
+                <span className="font-700" id="user-id-label">
+                    IdP Token:
+                </span>{' '}
+                <span aria-labelledby="user-id-label">{response?.idpToken}</span>
```

Comment:
Is it possible to pretty-format the raw JSON we receive from the backend? While it does work, it'd be great to have some sort of formatting of the string in a more prettier output, especially in case multiple claims are given (including things like sso.r.c. `realm-access.roles` which would be harder to read out right now).